docs/paper/dev/getting-started/paper-plugins.md

```diff
@@ -36,82 +36,91 @@ loader: io.papermc.testplugin.TestPluginLoader
 
 ### Dependency Declaration
 
-Dependency declaration is specified a bit differently than Bukkit plugins, as you are
-able to define if a dependency is required during bootstrapping. This means that any bootstrapping
-logic for your dependencies will be run before yours is run. 
+Paper Plugins change how to declare dependencies in your `paper-plugin.yml`:
+
 ```yml
 dependencies:
-  - name: DependencyName
-    required: true
-    bootstrap: true
+  bootstrap:
+    # Lets say that RegistryPlugin has some registry elements that this plugin requires.
+    # We don't need this during runtime, so it's not required in the server section. However
+    # can be added to both if needed
+    RegistryPlugin:
+      load: BEFORE
+      required: true
+      join-classpath: true # Defaults to true
+  server:
+    # Add a required "RequiredPlugin" dependency, which will load AFTER your plugin.
+    RequiredPlugin:
+      load: AFTER
+      required: true
+      # This means that your plugin will not have access to their classpath
+      join-classpath: false
 ```
 
-If a dependency is ``required`` it must be installed on the server or your plugin will fail to load. If marked otherwise, it will
-ignore if the dependency isn't found. 
-Otherwise, your plugin will be loaded fine, in the same fashion as a plugin that is a ``soft-depend`` in a Bukkit plugin.yml
-
-If a dependency is marked as ``bootstrap``, this indicates that this dependency is required during bootstrapping, which currently does not serve a purpose.
-
-:::note[Plugin Loading Order]
-
-Note that [unlike Bukkit plugins](#load-order-logic-split), dependencies are independent of the plugin load order. 
-This means that although declaring a dependency will give you access to its classes, if you require it to be initialised before your plugin is loaded, see how to [declare load order](#load-order-declaration).
-:::
-
-### Load Order Declaration
+With Paper Plugins, dependencies are split into two sections:
+- `bootstrap` - These are dependencies that you will be using in the [bootstrap](#bootstrapper).
+- `server` - These are dependencies that are used for the core functionality of your plugin, whilst the server is running.
 
-To declare the order in which plugins are loaded before/after your plugin, you are able to define two sections in your ``paper-plugin.yml``.
+Let's Take a look at a Dependency:
+```yml
+RegistryPlugin:
+  load: BEFORE # Defaults to OMIT
+  required: true # Defaults to true
+  join-classpath: true # Defaults to true
+```
 
-It should be noted that the load order for bootstrapping is independent, meaning that if you want to define the load order for bootstrappers
-you must set the ``bootstrap`` field to ``true``.
+- `load`: (`BEFORE`|`AFTER`|`OMIT`) Specifies whether this plugin should before or after **your** plugin.
+- `required`: Whether this plugin is required for your plugin to load.
+- `join-classpath`: Whether your plugin should have access to their classpath.
```

Comment:
What does having access to a classpath mean/why would you need to change this setting? Even the classpath section at the bottom doesn't *really* make obvious what this setting does